client/src/interface/User.ts

```diff
@@ -6,10 +6,12 @@ export interface Sitter {
 
 export interface User {
   _id?: string;
+  id?: string;
```

Comment:
I know you didn't add in the original `_id` but we have two ids here now which isn't necessary.

Reply:
Actually, I need this here without the _ because of the way the loggedInUser is passed as a User with no _ .